rtengine/rt_algo.cc

```diff
@@ -289,7 +382,7 @@ int calcContrastThreshold(float** luminance, float **blend, int W, int H) {
         }
     }
 
-    const float limit = (W - 2) * (H - 2) / 100.f;
+    const float limit = (W - 4) * (H - 4) / 100.f;
```

Comment:
Is this a raw-crop thing hardcoded? Or are W and H already influenced by the `border` parameter in the demosaicer?

Reply:
No. It's not related to a raw crop. It's because the algorithm needs a border. Have a look at the loop before (luminance[j][i+2] ...)